index.html

```diff
@@ -0,0 +1,260 @@
+                        <p class="programdescription">E-commerce, short for electronic commerce, refers to the buying and selling of goods and services over the internet. It has revolutionized the way businesses operate and how consumers shop.</p>
+                    </div>
+                    <div class="program">
+                        <img class="programimage" src="./images/program_icon_02 1.png" alt="program feature icon"/>
+                        <h2 class="programheader">Type of E-Commerce</h2>
+                        <p class="programdescription">E-commerce encompasses various types of businesses, including online marketplaces, retail stores, business-to-business (B2B) platforms, and digital service providers.</p>
+                    </div>
+                    <div class="program">
+                        <img class="programimage" src="./images/program_icon_03 1.png" alt="program feature icon"/>
+                        <h2 class="programheader">Forum</h2>
+                        <p class="programdescription">The key components of an e-commerce ecosystem include:Online Stores/Websites, Shopping Carts, Payment Gateways</p>
+                    </div>
+                    <div class="program">
+                        <img class="programimage" src="./images/program_icon_04 1.png" alt="program feature icon"/>
+                        <h2 class="programheader">Work Shop</h2>
+                        <p class="programdescription">E-commerce poses challenges such as security risks, competition, customer trust, and the need for effective digital marketing strategies to stand out in a crowded online marketplace.</p>
+                    </div>
+                    <div class="program">
+                        <img class="programimage" src="./images/program_icon_05 1.png" alt="program feature icon"/>
+                        <h2 class="programheader">Simulation</h2>
+                        <p class="programdescription">business-to-business (B2B) platforms, and digital service providers. These platforms provide an avenue for businesses to reach a global customer base and enable consumers to access a vast array of products and services from around the world.</p>
+                    </div>
+                </div>
+                <a class="LinkProgram" href="#">SEE THE HOLE PROGRAM</a>
+            </div>
+        </section>
+        <section class="SectionSpeakers" id="SectionSpeakers">
+            <div class="ContainerSectionSpeakers">
+                <div class="SectionSpeakT">
+                    Featured Speakers
+                </div>
+                <div class="RedLine"></div>
+                <div class="SpeakersCont" id="speakersContainer"></div>
+                <div class="SeeMoreButon">
+                    <a href="#" class="SeeMoreBtn BtnR" id="seeMoreBtn">
+                        See More
+                    </a>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] After the speaker section, we are supposed to have the `Partner` section and then have the `Footer` for the index page. This is currently missing. Kindly add these sections and please make sure you follow the given design.\

![image](https://github.com/jovasal/Capstone/assets/81932478/567505a6-c118-4b7e-ba85-1da5ccb7316c)

Reply:
- [ ] I was saying that this is a two-page website, it has the home page and the About page. And if you look at the given design they have a structure that involves both pages having a footer section, I was suggesting that you add the footer to the index page.